sdk/eventhub/azure-eventhubs/azure/eventhub/common.py

```diff
@@ -152,27 +153,50 @@ def _trace_link_message(self, parent_span=None):
                 if traceparent:
                     current_span.link(traceparent)
 
+    def _get_runtime_info(self):
```

Comment:
Let's change this method name to have last_enqueued_event_properties instead of "runtime_info" so code readers can link them easily.

Reply:
updated